Foundation/cmake/PocoFoundationConfig.cmake

```diff
@@ -1 +1,8 @@
-include("${CMAKE_CURRENT_LIST_DIR}/PocoFoundationTargets.cmake")
\ No newline at end of file
+include("${CMAKE_CURRENT_LIST_DIR}/PocoFoundationTargets.cmake")
+
+if (@POCO_UNBUNDLED@)
+    if (@HUNTER_ENABLED@)
+        hunter_add_package(ZLIB)
```

Comment:
`hunter_add_package` not needed here.

Reply:
Thanks for the review. 
Removed: https://github.com/hunter-packages/poco/pull/1/commits/056037f3dbcdf9ba5a2bbde3fd219e4f521df7ea

Just to make sure my understanding is correct, `hunter_add_package` is not required here because its already been called once inside `hunter_add_package(PocoCpp)`?

Reply:
> Just to make sure my understanding is correct, hunter_add_package is not required here because its already been called once inside hunter_add_package(PocoCpp)?

Yes. `hunter_add_package(PocoCpp)` will download PocoCpp archive with sources, unpack it and run CMake configure step. While configure step we will call `hunter_add_package` for all dependencies like `hunter_add_package(ZLIB)`.